lib/ReactGridLayout.jsx

```diff
@@ -250,10 +250,18 @@ export default class ReactGridLayout extends React.Component {
 
     // Set state
     this.setState({
-      layout: compact(layout, this.props.verticalCompact),
       activeDrag: null,
       oldDragItem: null
     });
+
+    // Set State and callback, but only if layout changed
+    const newLayout = compact(layout, this.props.verticalCompact);
+    if (!isEqual(newLayout, this.state.layout)) {
```

Comment:
Additionally, since `onLayoutChange` is *only* called on `Stop` handlers, this means that if the layout didn't happen to change in that very last movement, the callback won't fire.

I'll have to revert this piece.